src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
```

Comment:
Using the `*` selector is not recommended as it impacts performance. Try to target specific elements that need the style.